app/routes/languages.rb

```diff
@@ -6,9 +6,11 @@ class Languages < Core
       TOPICS = [:about, :exercises, :installing, :tests, :learning, :resources, :help, :launch].freeze
 
       get '/languages' do
-        tracks = X::Track.all
-        active, inactive = tracks.partition(&:active?)
-        planned, inactive = inactive.partition(&:planned?)
+        tracks = Trackler.tracks
+        active, inactive = tracks.partition { |t| t.active? }
+        planned = []
```

Comment:
@kytrinyx @mhelmetag Would like some insight from either of you about how planned tracks should be implemented. This is just a placeholder to make tests pass.